src/components/common/ItemsNotFoundComponent.jsx

```diff
@@ -11,6 +11,8 @@ const ItemsNotFoundComponent = props => (
       <h2>Nothing To See Here!</h2>
       <Image centered src="images/windEmpty.jpg" alt="empty box" id="not_found_image" />
       <p> {props.message} </p>
+      {props.children}
```

Comment:
Since we are not passing any children to this component in `UserComponent.jsx`,  this change can be deleted